index.html

```diff
@@ -388,7 +388,7 @@
             <div class="Companies-item">
               <div class="Company Company--jc">
                 <div class="Company-brand">
-                  <a class="Company-brandLink" href="http://jamescooke.info" rel="external">
+                  <a class="company-brandlink" href="http://jamescooke.info/pages/coding-cooke.html" rel="external">
```

Comment:
Can you change this back to a capital `C` at the start of the CSS class please - it follows the [SuitCSS](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#ComponentName) naming conventions